pkg/secrethub/account_key.go

```diff
@@ -49,7 +49,7 @@ func (s accountKeyService) Create(verifier credentials.Verifier, encrypter crede
 // Exists returns whether an account key exists for the client's credential.
 func (s accountKeyService) Exists() (bool, error) {
 	_, err := s.client.getAccountKey()
-	if err == api.ErrAccountKeyNotFound || err == api.ErrCredentialNotKeyed {
```

Comment:
I believe `api.ErrCredentialNotKeyed` is a 409 (conflict), not a not found error.

Reply:
When does `api.ErrCredentialNotKeyed` occur? I'm hoping only on a POST to e.g. credentials?

Reply:
> I believe `api.ErrCredentialNotKeyed` is a 409 (conflict), not a not found error.

This is incorrect; `api.ErrCredentialNotKeyed` is a not found error.

It is returned when the account is keyed (has a public account key) but there is no private key stored that is encrypted for the credential with which the request is authenticated.

So, the proposed change provides the desired behavior :+1: